extensions/interactions/rule_templates.json

```diff
@@ -276,5 +276,16 @@
     "FuzzyEquals": {
       "description": "is equal to {{x|NormalizedString}}, misspelled by at most one character"
     }
+  },
+  "RatioExpressionInput": {
+    "Equals": {
+      "description": "is equals to {{x|RatioExpression}}"
```

Comment:
``is equal to`` similar to other equals rules.

Reply:
done